### PR TITLE
[Testing] Add three test methods

### PR DESCRIPTION
**Pull request by ghost**
- modPowTest
- modInverseTest
- JacobiTest

### CODE REVIEW

File: BigIntegerTests/Methods.cs [hunk 1]
```diff
@@ -2,12 +2,12 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.Text;
 using System.Collections.Generic;
```
Comment (bazzilic) [inline BigIntegerTests/Methods.cs]: Before making a pull request, please remove unused `using` directives

File: BigIntegerTests/Methods.cs [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
-
```
Comment (bazzilic) [inline BigIntegerTests/Methods.cs:5]: Also, leave blank lines for better readability

File: BigIntegerTests/Methods.cs [hunk 3]
```diff
@@ -79,5 +79,64 @@ public void TestGetBytes()
...
+            }
+        }
+
+        [TestMethod()]
+        public void JacobiTest()
+        {
+            // Value generated from http://math.fau.edu/richman/jacobi.htm
+            int[,] jcbTbl = { 
+                { 1236, 20003, 1 }, 
+                { 4513022, 3436547, -1 },
```
Comment (bazzilic) [inline BigIntegerTests/Methods.cs]: There are trailing spaces in these lines. In Visual Studio you can do automatic cleanup of this kind of things using `Ctrl+K, Ctrl+F` after selecting parts you want to fix. It fixes trailing spaces, indentation and spaces between symbols.

File: BigIntegerTests/Methods.cs [hunk 4]
```diff
@@ -79,5 +79,64 @@ public void TestGetBytes()
                 Assert.AreEqual(zero_bytes[0], (byte)0);
             }
         }
+
+
+        [TestMethod()]
+        public void modPowTest()
+        {
+            BigInteger a = new BigInteger("4513022378190195207248111493619814210011122111521314021116172245292421892189133135249253284210917322371331631915863149241442281401995510735118116112172202199102116124234501111031274954151507124570516154178228146", 10);
```
Comment (bazzilic) [inline BigIntegerTests/Methods.cs]: Please use `var` where applicable.

File: BigIntegerTests/Methods.cs [hunk 5]
```diff
@@ -79,5 +79,64 @@ public void TestGetBytes()
...
+            Assert.AreEqual(1814, b.LongValue());
+
+            try
+            {
+                b = a.modInverse(new BigInteger(9999998));
+            }
+            catch (ArithmeticException)
+            {
+                Assert.IsTrue(true);
+            }
```
Comment (bazzilic) [inline BigIntegerTests/Methods.cs:117]: Lines 110-117: could you explain what happens here? From what I see, these lines don't do anything at all.

File: BigIntegerTests/Methods.cs [hunk 6]
```diff
@@ -79,5 +79,64 @@ public void TestGetBytes()
                 Assert.AreEqual(zero_bytes[0], (byte)0);
             }
         }
+
+
+        [TestMethod()]
+        public void modPowTest()
```
Comment (bazzilic) [inline BigIntegerTests/Methods.cs]: Please, maintain the naming style used in the project. That should be `TestModPow`.

File: BigIntegerTests/Methods.cs (hunk 5 above)
Comment (ghost) [reply BigIntegerTests/Methods.cs:117]: @bazzilic: BigInteger _a_ and _9999998_ are even, so _modInverse_ should raise an exception.

Comment (bazzilic) [reply BigIntegerTests/Methods.cs:117]: Yes, but in this case if it _does not_ raise an exception, the test will still pass. Should be smth like

``` csharp
var exceptionRaised = false;
try
{
    b = a.modInverse(new BigInteger(9999998));
}
catch (ArithmeticException)
{
    exceptionRaised = true;
}
Assert.IsTrue(exceptionRaised);
```

Comment (ghost) [reply BigIntegerTests/Methods.cs:117]: You're right! Will correct it.

**Pull request merged**